internal/controller/ipam/adapter/infoblox_adapter.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2025
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adapter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/telekom/cluster-api-ipam-provider-infoblox/api/v1alpha1"
```

Comment:
please rename the package alias, otherwise `v1alpha1` tells nothing and worsens the readability